src/Digdir.Domain.Dialogporten.GraphQL/EndUser/DialogById/Subscriptions.cs

```diff
@@ -0,0 +1,36 @@
+using AutoMapper;
+using Digdir.Domain.Dialogporten.Application.Features.V1.EndUser.Dialogs.Queries.Get;
+using Digdir.Domain.Dialogporten.GraphQL.Common.Authorization;
+using HotChocolate.Authorization;
+using MediatR;
+using Constants = Digdir.Domain.Dialogporten.Infrastructure.GraphQl.GraphQlSubscriptionConstants;
+
+// ReSharper disable ClassNeverInstantiated.Global
+
+namespace Digdir.Domain.Dialogporten.GraphQL.EndUser.DialogById;
+
+[Authorize(Policy = AuthorizationPolicy.EndUser)]
+public sealed class Subscriptions
+{
+    [Subscribe]
+    [Topic($"{Constants.DialogUpdatedTopic}{{{nameof(dialogId)}}}")]
+    public async Task<DialogByIdPayload> DialogUpdated(Guid dialogId,
+        [Service] ISender mediator,
+        [Service] IMapper mapper,
+        [EventMessage] Guid eventId,
+        CancellationToken cancellationToken)
+    {
+        if (dialogId != eventId)
```

Comment:
1. What if they don't match? (Something has gone horribly wrong with HotChocolate/Redis)

2. Do we need to check ownership/validity of the dialogId when starting a subscription?
(We do get that check once we call GetDialogQuery when attempting to return data, with possible error messages "notFound", "deleted", or "forbidden")